cookbooks/aws-parallelcluster-slurm/recipes/config_head_node.rb

```diff
@@ -38,11 +38,21 @@
   mode '0700'
 end
 
+# slurm node bootstrap timeout
+config = YAML.load_file(node['cluster']['cluster_config_path'])
+timeouts = config.dig("DevSettings", "Timeouts")
+timeout = if timeouts && !timeouts.empty?
+            timeouts["ComputeNodeBootstrapTimeout"]
+          else
+            1800
+          end
+
```

Comment:
For now shall we propagate the timeout in extra_json so that we don't depend on DevSettings and also we always set the default from the CLI?

Reply:
move the value to extra_json, but I also set a default in default.rb `default['cluster']['compute_node_bootstrap_timeout'] = 1800`